### DISCUSSION TITLE
Cathode aging mechanisms

### DISCUSSION THREAD

**Discussion by fsenoner** [pybamm-team/PyBaMM#3504] [Q&A] [answered]
Dear PyBaMM team,
Dear @DrSOKane,

I was wondering if there are any aging mechanisms implemented in PyBaMM for the cathode, apart from the ones related to the Anode such as SEI and Li-plating. I have gone through the documentation (https://docs.pybamm.org/en/latest/source/api/models/base_models/base_battery_model.html#) but could only find the particle mechanics submodel, which allows for swelling and cracking to be modeled in the cathode. Is there any feature that enables the modeling of rock salt formation in the cathode?

Thank you for your help and assistance.
Fabian

**Comment by DrSOKane** [comment, marked as answer]
Hi Fabian,

My former colleague, Dr. Mingzhao Zhuo, wrote a custom version of PyBaMM in which the rock salt phase transition was implemented. You can read about his findings in [this paper](https://doi.org/10.1016/j.jpowsour.2022.232461), with the source code being available [here](https://github.com/mzzhuo/PE_degradation). Unfortunately, Mingzhao's code is incompatible with the particle mechanics submodels in PyBaMM, so it is not part of public-facing PyBaMM.

However, PyBaMM is always interested in learning more about positive electrode degradation. If you have any published data on positive electrode degradation, we would love to see it.

Many thanks,
Dr. Simon O'Kane
Research Associate, Imperial College London

**Reply by fsenoner** [reply]
Thank you for your response, Dr. O'Kane.

It's great to hear that there is a custom version of PyBaMM available that includes the rock salt phase transition. Thank you also for providing the reference to the study and the link to the source code.
Are there any plans to incorporate a similar rock salt phase transition in PyBaMM in the future?

Regarding my question, I was wondering if the current version of PyBaMM also models an increase in resistance for the cathode alongside the swelling and cracking settings in the particle mechanics submodel.

Thank you in advance for your response.
Best regards,
Fabian Senoner

**Reply by DrSOKane** [reply]
Hi Fabian,

Sorry for the delay in responding. The degradation modelling team at Imperial has been pondering this very question, and we still don't have an answer. All I can say at this point is a positive electrode degradation model, that is not yet implemented in PyBaMM, is within the scope of a publication we are working on.

Many thanks, Simon

**Reply by fsenoner** [reply]
Hi Simon,

Thank you very much for your response. In the mean time I will use the PyBaMM branch pe_degradation from Dr. Mingzhao Zhuo.

Best regards,
Fabian Senoner
